src/main/java/Duke.java

```diff
@@ -1,10 +1,245 @@
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.time.LocalTime;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.File;
+
 public class Duke {
     public static void main(String[] args) {
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("What can I do for you?\n");
+
+        boolean isOnline = true;
+        ArrayList<Task> taskList = new ArrayList<>();
+
+        File savedTasks = new File("saved_tasks_list.txt");
+        try {
+            if (!savedTasks.createNewFile()) {
+                // Load the exist file
+                Scanner fileScanner = new Scanner(savedTasks);
+                while (fileScanner.hasNextLine()) {
+                    String record = fileScanner.nextLine();
+                    String[] fields = record.split("\\|");
+                    Task currTask;
+                    if (fields[0].equals("T")) {
+                        currTask = new Todo(fields[2]);
+                    } else if (fields[0].equals("D")) {
+                        currTask = new Deadline(fields[2], fields[3]);
+                    } else if (fields[0].equals("E")) {
+                        currTask = new Event(fields[2], fields[3], fields[4]);
+                    } else {
+                        currTask = new Task("");
```

Comment:
What would happen if text file was corrupted and fields[0] traversed through here? Should it still create the empty task?